src/app/pages/backend-integration/diagram/backend-integration-diagram.component.html

```diff
@@ -0,0 +1,81 @@
+<div class='flex-column justify-content-center'>
```

Comment:
Please, use double quotes in the templates.

Reply:
Linter is showing errors for double quotes, i think it's missconfiguration in project settings